pkgs/development/python-modules/guestfs/default.nix

```diff
@@ -11,6 +12,9 @@ buildPythonPackage rec {
   version = "1.40.1";
   format = "setuptools";
 
+  # undefined symbol: PyEval_CallObject
```

Comment:
I get
```
actions-0.c: In function ‘guestfs_int_py_acl_delete_def_file’:
actions-0.c:54:7: error: implicit declaration of function ‘PyEval_ThreadsInitialized’ [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wimplicit-function-declaration-Wimplicit-function-declaration8;;]
   54 |   if (PyEval_ThreadsInitialized ())
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~
```
and don't see `PyEval_CallObject`. Is that non-deterministic?

Reply:
`PyEval_CallObject` is the error when running on NixOS 24.11.20250225.5d7db46 but I do get `PyEval_ThreadsInitialized` when ruining on master.